android/app/src/debug/AndroidManifest.xml

```diff
@@ -4,7 +4,6 @@
          to allow setting breakpoints, to provide hot reload, etc.
     -->
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
```

Comment:
that line should not be removed:
<uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />